identityClient/api.go

```diff
@@ -207,3 +207,15 @@ type CompleteChallengeRequest struct {
 	SignedChallenge string `json:"signed_challenge"` // signed challenge in the form of "{challenge}{timestamp}"
 	SignedTime      int64  `json:"sign_time"`        // time challenge was signed
 }
+
+type LDAPCache struct {
```

Comment:
comments for any public module exports please, adds documentation for us and future users (especially now that e3db-clients-go is fully exposed via e3db-go)

Reply:
Will do